notifications/http_test.go

```diff
@@ -122,7 +122,7 @@ func TestHTTPSink(t *testing.T) {
 		},
 		{
 			// Case where connection never goes through.
-			url:     "http://shoudlntresolve/",
+			url:     "shoudlntresolve",
```

Comment:
Yes, the original test is using to cover the case where the connection cannot be  established because it was expected to  fail to parse the url-`http://shouldnotresolve/` and return a http.StatusBadRequest. But if the node where we run this test is behind some proxy, it is possible that the proxy will check its white list first and then return a `403-forbidden` as the response if the url is not on the list.

I don't feel strongly about this case, for it's up to the implementation of the proxy.